code/game/objects/items/devices/taperecorder.dm

```diff
@@ -242,11 +233,14 @@
 			return
 		else if(playing == 1)
 			playing = 0
-			var/turf/T = get_turf(src)
-			for(var/mob/O in hearers(world.view-1, T))
-				O.show_message("<font color=Maroon><B>Tape Recorder</B>: Playback stopped.</font>",2)
+			recorder_message("Playback stopped")
 			icon_state = "taperecorderidle"
 			return
 		else
 			usr << "<span class='warning'>Stop what?</span>"
-			return
\ No newline at end of file
+			return
+
+/obj/item/device/taperecorder/proc/recorder_message(var/msg)

```

Comment:
Rather than making this !SNOWFLAKE!, I suggest adding an audible_message() proc similar to the current visible_message one for this sort of behaviour.